blogs/2020/05/06/github-issues-integration.md

```diff
@@ -0,0 +1,65 @@
+
+![User Hover](user-hover.png)
+
+Issue context is typically needed when commiting a change that resolves a particular issue, within code files, and in markdown (such as a changelog). To easily add this context, we have completion suggestions for issues and users in all of these places. In the commit textbox, you can format your issue completion with `"githubIssues.issueCompletionFormatScm"`. In markdown, issues complete as a markdown link. In other files, issues complete as a simple issue number (#1234).
+
+![Completion Suggestions](completion-suggestions.gif)
+
+The list of possible issues is configurable with the setting `"githubIssues.queries"`, so if you work accross multiple repositories you can include queries for those issues. The list of users are the collaborators in the currently open repository.
+
+![Issue Queries](issue-queries.png)
+
+## Creating an Issue from Anywhere
+
+When we find a bug in VS Code while working on some source code, we create an issue and assign it to whoever owns that area. Or, if the finder of the bug is also the owner, we'll often leave a //TODO comment as a reminder to come back to it. Having //TODOs scattered throughout code is hard to keep track of when you have many contributors (though it's safe to say that we've all done it), but creating an issue and referencing it in the //TODO is trackable. To reduce the barrier and context loss for creating an issue while you're deep in the source there are a few new ways to create issues:
+
+From a //TODO comment (configurable with `githubIssues.createIssueTriggers`), you can create and assign an issue without leaving VS Code.
+
+![Create Issue from TODO](create-from-todo.gif)
+
+And from a selection, you can quickly create an issue with a permalink back to the code it originated from using the **GitHub Issues: Create Issue from Selection** command. If you just need a pointer to some code, you can also use the **GitHub Issues: Copy GitHub Permalink** command. Finally, if there is failure information in the terminal, you can just copy the output to the clipboard and create an issue using **GitHub Issues: Create Issue from Clipboard**.
+
+## Working on Issues
+
+A common workflow is to look at your issues, pick one to work on, create a branch to work in, make some commits, then merge your changes back into master with a pull request. From the new Issues view you can do exactly that.
+
+![Work on an Issue](work-on-issue.gif)
+
+To fit more workflows there are several options you can configure. If your flow doesn't involve creating a topic branch, you can disable the branch creation with `"githubIssues.useBranchForIssues"`. If you have a different naming scheme for your branches you can use the `"githubIssues.workingIssueBranch"` setting. The issues listed in the Issues view can be configured to use a custom query with `"githubIssues.queries"`.
```

Comment:
regarding queries - i might suggest that the query syntax is the same syntax you use in GitHub, and maybe give an example